apps/nextjs/src/components/AppComponents/Layout/index.tsx

```diff
@@ -15,15 +15,17 @@ const Layout = ({
 }) => {
   const isDemoUser = useDemoUser().isDemoUser;
   return (
-    <div className="flex min-h-screen flex-col ">
-      <Header />
-      <main
-        className={`flex h-full flex-1 flex-col bg-muted/50 bg-white ${isDemoUser && "pt-28 sm:pt-20"}`}
-      >
-        <DialogProvider>{children}</DialogProvider>
-      </main>
-      {includeFooter && <Footer />}
-    </div>
+    <DialogProvider>
```

Comment:
Any danger that we end up with nested DialogProviders? I thought this was in the layout.tsx file